lib/plugins/create/templates/aws-nodejs/serverless.yml

```diff
@@ -101,3 +79,15 @@ functions:
 #     NewOutput:
 #       Description: "Description for the output"
 #       Value: "Some output value"
+
+
+##───────────────────────────────────────────────────────────────────────
+## 'package' - Include or Exclude artifacts in your service http://bit.ly/2rJBhU9
```

Comment:
Good call. What glob syntax do we use? https://github.com/sindresorhus/multimatch ?